poetry.toml

```diff
@@ -0,0 +1,2 @@
+[virtualenvs]
+in-project = true
```

Comment:
Not sure if this file should be added to .gitignore or not, I've never used poetry before 😓

Reply:
I _think_ this file is only responsible for local configuration but I'm not sure